src/UserRepository.js

```diff
@@ -1,11 +1,41 @@
-import sleepData from './data/sleep';
-import hydrationData from './data/hydration';
-import activityData from './data/activity';
-import userData from './data/users';
+
+import User from './User'
 
 class UserRepository {
-  constructor(date, userData, activityData, hydrationData, sleepData) {
-    this.users = [];
+  constructor(rawData, todaysDate) {
```

Comment:
good idea of consolidating user-, activity-, hydration-, & sleepData into one single parameter (`rawData`)